aws-ami-creation/hcl2-files/build.pkr.hcl

```diff
@@ -0,0 +1,17 @@
+build {
+  name = "build-hf-dl-neuron"
+  sources = [
+    "source.amazon-ebs.ubuntu"
+  ]
+  // provisioner "shell" {
+  //   inline = [
+  //     "echo foo"
+  //   ]
+  // }
```

Comment:
I assume this is a leftover of previous attempts. If so consider removing it.

Reply:
It's already removed :)